Dockerfile

```diff
@@ -143,7 +148,7 @@ HEALTHCHECK --interval=5m --timeout=3s \
 
 # Set up the 'atlantis' user and adjust permissions
 RUN addgroup atlantis && \
-    adduser -S -G atlantis atlantis && \
+    adduser -u 100 -S -G atlantis atlantis && \
```

Comment:
We need to discuss this.
We need to keep backward compatibility or make the breaking change.
A lot of people is already using this and if we change it it could bring some deployments down

Reply:
@GMartinez-Sisti @nitrocode @GenPage

Reply:
The atlantis user does not need a shell to start, so a system user was used.

Reply:
I'm not saying this is wrong but changing it will break backwards compatibility

Reply:
A possible option is to add both users (100 and 1000) on debian. This way the helm chart will be correct and existing users won't have issues. Otherwise we will need to do a breaking change release indeed. Thoughts?

Reply:
that could work

Reply:
Looks like two maintainers agreed, so I'd say let's go with this. @kvanzuijlen can you please update it?

Reply:
Sounds good! I'll have a look

Reply:
This is for alpine and should stay on 100, as it already was the default